### DISCUSSION TITLE
Community help

### DISCUSSION THREAD

**Discussion by vplemyannik** [keycloak/keycloak#19505] [Ideas]
Hi, i want to help with bugs/features that have label `help wanted`. Is it a good idea to start discussion in the thread of the issue or do i need to notify all in google group https://groups.google.com/g/keycloak-dev?  
It seems to me that good way is start the discussion inside issue if the fix is clear and small. Is there someone who i can mention to ask questions or discuss with?

**Comment by jonkoops** [comment]
If you are looking to sign up for issues feel free to comment in the issue itself. If you have a clear idea of what to do we can assign the issue to you.
